### PR TITLE
JN-372 sonar-scanner

### PR DESCRIPTION
**Pull request by TomConner**
Run SonarCloud across api-admin, api-participant, core, ui-admin, ui-participant. Using sonar-scanner rather than gradle because this repo contains both Java and TypeScript.

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+sonar.projectKey=broadinstitute_pearl
```
Comment (nawatts) [inline sonar-project.properties:1]: Will this need to change if we rename the repo?

**Review by nawatts — APPROVED** [review]

File: sonar-project.properties (hunk 1 above)
Comment (devonbush) [reply sonar-project.properties:1]: I talked to Tom about this -- some stuff will need to change, but it shouldn't be a big deal

**Review by devonbush — APPROVED** [review]

File: sonar-project.properties (hunk 1 above)
Comment (TomConner) [reply sonar-project.properties:1]: ^^ that's right

**Comment by TomConner** [comment]
merging to get sonar workflow

**Pull request merged**